uproot/write/objects/TH.py

```diff
@@ -270,7 +270,7 @@ def length_tattaxis(self):
     def return_taxis(self, cursor, axis):
         cnt = numpy.int64(self.length_taxis(axis) - 4) | uproot.const.kByteCountMask
         vers = 10
-        if axis["_fLabels"] or axis["_fModLabs"]:
+        if axis.get("_fLabels", None) != [] or axis.get("_fModLabs", None) != []:
```

Comment:
Wait a minute—if the null condition is that `axis["_fLabels"] == []` (same for `_fModLabs`), then you need to do

```python
if axis.get("_fLabels, []) != [] or axis.get("_fModLabs", []) != []
```

and if you want to be agnostic about whether it's a list, a tuple, or something else, you can do

```python
if len(axis.get("_fLabels", [])) > 0 or len(axis.get("_fModLabs", [])) > 0
```

Be sure you understand what `axis.get` is doing before using it in code.